lib/engine/game/g_1858/step/track.rb

```diff
@@ -10,11 +10,17 @@ class Track < Engine::Step::Track
           def actions(entity)
             return [] unless entity == current_entity
             return [] if !entity.minor? && !entity.corporation?
```

Comment:
Do you have operating round entities that aren't minors or corporations? If not, this line is unnecessary.

Reply:
No, you're right. I was being overly cautious/paranoid. I've removed this line.